krun/platform.py

```diff
@@ -914,15 +917,17 @@ def check_preliminaries(self):
 
         if self.config.ENABLE_PINNING:
             self._check_cset_installed()
+        # Clear any leftover cpu pinning since it will break taskset
+        self.clear_cpu_pinning()
```

Comment:
Should we remove the `ENABLE_PINNING` check in `clear_pinning` and move this code into the above branch?

Reply:
i guess so now.